runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -68,21 +70,29 @@
 	private static final int BORDER = 5;
 	private static final int BORDER_TOP = BORDER + 15;
 	private static final int PADDING = 2;
+	private static final int MIN_OVERLAY_SIZE = 32;
+	private static final int OVERLAY_RESIZE_TOLERANCE = 10;
```

Comment:
It might be good to drop this a few px, its easy to get the resize zone on small overlays when you want to move it.